File: CarlaErik/UvA.SoftCon.Questionnaire/AST/Types/BoolValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UvA.SoftCon.Questionnaire.AST.Types
{
    public class BoolValue : Value<bool>
    {
        public BoolValue(bool value)
            : base(value) { }
    }
}

File: Nian_Karl/SoftwareConstruction/src/org/uva/qls/ast/value/BoolValue.java
package org.uva.qls.ast.value;


public class BoolValue extends Value  {

	private final Boolean value;

	public BoolValue(Boolean value) {
		this.value = value;
	}

	@Override
	public Boolean value() {
		return value;
	}

	@Override
	public String toString() {
		return value.toString();
	}
	
	@Override
	public boolean isDefined() {
		return true;
	}
}
